src/Model/PhasorDynamics/SynchronousMachine/GENROUwS/Genrou.cpp

```diff
@@ -14,6 +14,7 @@
 #include <iostream>
 
 #include <Model/PhasorDynamics/Bus/Bus.hpp>
+#include <Model/PhasorDynamics/Governor/Tgov1/Tgov1.hpp>
 #include <Model/PhasorDynamics/SynchronousMachine/GENROUwS/GenrouData.hpp>
```

Comment:
I would mark this as temporary, so we don't forget about this. Consider adding a comment like this:
```suggestion
#include <Model/PhasorDynamics/Governor/Tgov1/Tgov1.hpp> // <- TODO: Temporary, to be removed.
#include <Model/PhasorDynamics/SynchronousMachine/GENROUwS/GenrouData.hpp>
```